pom.xml

```diff
@@ -64,5 +90,55 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-cps</artifactId>
+            <version>${workflow-cps.version}</version>
+            <optional>true</optional>
```

Comment:
same here - you do not seem to need to depend on this, you just want an optional on workflow job do you not?

Reply:
`hudson.tasks.junit.pipeline.JUnitResultsStepExecution` uses `org.jenkinsci.plugins.workflow.cps.nodes.StepStartNode`